privacy.md

```diff
@@ -17,7 +17,7 @@ When it comes to your personal information you may have the right to:
 - File a complaint with us or a Data Protection Authority
 
 Some of the above rights can be exercised yourself through IRC Services,
-see `/msg NickServ HELP`, with a focus on the commands `SET`, `SET TAXONOMY`
+see `/msg NickServ HELP`, with a focus on the commands `SET`, `TAXONOMY`
```

Comment:
"SET TAXONOMY" is already pretty much covered by "SET"; I think the intention here was to document the "TAXONOMY" command to *fetch* metadata, so changing it to "SET TAXONOMY" (a) is a bit pointless as the overall "SET" is already there, (b) removes the mention of using e.g. `/msg NickServ TAXONOMY biggpresh` to *read* metadata.

Now, I would argue that this list should include "INFO", for fetching info on a nick (i.e. between INFO and TAXONOMY a user can see what information services holds on them.

Reply:
@bigpresh My intention was to change `SET TAXONOMY` -> `TAXONOMY` as `TAXONOMY` isn't valid sub-command of `SET`. Yeah `INFO` could be mentioned too I guess.